core/src/test/java/org/apache/iceberg/avro/TestAvroSchemaProjection.java

```diff
@@ -150,4 +152,16 @@ public void projectWithMapSchemaChanged() {
         .as("Result of buildAvroProjection is missing some IDs")
         .isFalse();
   }
+
+  @Test
+  public void testVariantConversion() {
```

Comment:
Is the projection test suite the right place for conversion tests? Isn't there a conversion test suite?

Reply:
You are right. Let me move there.